### ISSUE TITLE
[BITV] The link to copy the RSS URL does not have an alternative text.

### ISSUE THREAD

**Issue by Pytal** [nextcloud/activity#1087]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v5-n1

![image](https://user-images.githubusercontent.com/24800714/218900883-51a97bd0-efae-4720-ac97-4e93749de0b7.png)

</details>
